modules/core/src/main/scala/org/tessellation/cli/method.scala

```diff
@@ -71,8 +71,8 @@ object method {
       peerDiscoveryDelay = PeerDiscoveryDelay(
         checkPeersAttemptDelay = 1.minute,
         checkPeersMaxDelay = 10.minutes,
-        additionalDiscoveryDelay = 3.minutes,
-        minPeers = 2
+        additionalDiscoveryDelay = 0.minutes,
```

Comment:
Aren't we effectively disabling this whole functionality? If so shouldn't we just remove/revert it (the delay functionality)? There is no point in maintaining obsolete not used code.

Reply:
Per our offline discussion, we will proceed with disabling this functionality for now.